### DISCUSSION TITLE
Netbox report error

### DISCUSSION THREAD

**Discussion by trs91km** [netbox-community/netbox#12560] [General]
I tried to upload one report, after that when I tried to access report tab, the portal got broken with below error, please help to fix this

<class 'SyntaxError'>

invalid syntax (netbox_device roles.csv, line 1)

Python version: 3.10.6
NetBox version: 3.5.1

**Comment by candlerb** [comment]
In Netbox, "Reports" are scripts written in Python that you write to extract data from the database - not CSV files.

See https://docs.netbox.dev/en/stable/customization/reports/
